example/deploy/role.yaml

```diff
@@ -171,31 +171,76 @@ metadata:
   creationTimestamp: null
   name: network-operator
 rules:
-- apiGroups:
-  - mellanox.com
-  resources:
-  - '*'
-  verbs:
-  - create
-  - delete
-  - get
-  - list
-  - patch
-  - update
-  - watch
-- apiGroups:
+  - apiGroups:
+    - rbac.authorization.k8s.io
+    resources:
+    - roles
+    - rolebindings
+    verbs:
+    - '*'
+  - apiGroups:
     - ""
-  resources:
-    - nodes
-  verbs:
+    resources:
+    - pods
+    - services
+    - services/finalizers
+    - endpoints
+    - persistentvolumeclaims
+    - events
+    - configmaps
+    - secrets
+    - serviceaccounts
+    verbs:
+    - create
+    - delete
     - get
     - list
+    - patch
+    - update
     - watch
-- apiGroups:
+  - apiGroups:
     - apps
-  resources:
-    - daemonsets
-  verbs:
+    resources:
```

Comment:
do we need to be able to CRUD all resources under apps ?
can this be limited to a subset ?

Reply:
As a first pass I likely added more operations under apps in the interest of expedience, I can narrow them down.